pkg/network/tracer/tracer_test.go

```diff
@@ -55,7 +55,11 @@ var (
 const runtimeCompilationEnvVar = "DD_TESTS_RUNTIME_COMPILED"
 
 func TestMain(m *testing.M) {
-	log.SetupLogger(seelog.Default, "warn")
+	logLevel := os.Getenv("LOG_LEVEL")
```

Comment:
for consistency, maybe?
```suggestion
	logLevel := os.Getenv("DD_LOG_LEVEL")
```